imports/stylesheets/incidents/incidents.import.styl

```diff
@@ -28,15 +28,12 @@
     max-width 800px
     margin 30px auto
   .modal-content
-    display flex
     flex-direction column
-    height calc(100vh - 60px)
-    min-height 400px
+  .modal-header
   .modal-body
-    padding 0
-    position relative
+    padding 0 0 0 10px
```

Comment:
Lets also add some padding to the right side so the text has a little breathing room.